tox.ini

```diff
@@ -10,6 +10,7 @@ env_list =
     3.10
     3.9
     3.8
+    graalpy
```

Comment:
No version constraint here? graalpy-3.13, graalpy-3.12?

Reply:
It's just like `pypy3` (except that graalpy only targets Python 3, we don't have a Python 2 variant like PyPy does so no `3` suffix)